src/pages/List/HotelList.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export const HotelList = props => {
```

Comment:
```suggestion
export const HotelList = ({image,content, ~~~}) => {
```
매개변수 자리에서 바로 구조분해할당 가능합니다!

Reply:
수정하였습니다.